File: jniDemo/src/main/java/com/megvii/jnidemo/MainActivity.java
package com.megvii.jnidemo;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import com.megvii.jnidemo.handler.NativeWorker;

public class MainActivity extends AppCompatActivity {
    private NativeWorker nativeWorker;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        initData();
    }

    private void initData() {
        nativeWorker = new NativeWorker();
        nativeWorker.sayHello();
    }
}

File: app_kotlin/src/main/java/com/example/app_kotlin/MainActivity.kt
package com.example.app_kotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.whkf.st.HelloWorld

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val multiplyValue = HelloWorld().multiply(5, 6);
    }
}